tgstation.dme

```diff
@@ -3908,4 +3908,5 @@
 #include "modular_skyrat\modules\verbs\code\modules\mob\say.dm"
 #include "modular_skyrat\modules\verbs\code\modules\mob\subtle.dm"
 #include "modular_skyrat\modules\xenomorph\code\xenomorph_balance.dm"
+#include "modular_skyrat\organs\lungs.dm"
```

Comment:
You're gonna need to put the files in the modules folder, see above

Reply:
doing it

Reply:
done